packages/grafana-schema/src/schema/dashboard/v2alpha0/examples.ts

```diff
@@ -222,18 +276,42 @@ export const handyTestingSchema: DashboardV2Spec = {
               kind: 'ElementReference',
               name: 'panel-2',
             },
-            height: 100,
+            height: 10,
             width: 200,
             x: 0,
             y: 2,
           },
         },
+        {
+          kind: 'GridLayoutRow',
+          spec: {
+            y: 20,
+            collapsed: false,
+            title: 'Row 1',
+            repeat: { value: 'customVar', mode: 'variable' },
+            elements: [
+              {
+                kind: 'GridLayoutItem',
+                spec: {
+                  element: {
+                    kind: 'ElementReference',
+                    name: 'panel-3',
+                  },
+                  height: 10,
+                  width: 10,
+                  x: 0,
+                  y: 0,
```

Comment:
@oscarkilhed - have you captured anywhere that x/y for row elements is now relative to the row? I think we can document this in schema code comment and also as in https://docs.google.com/document/d/1pEjy3wpTBhlFh_02L-TvmNuLVE8mZRYMZH67Ipux-oc/edit?tab=t.gmfnl57ofc1z#heading=h.4cdo1fkrz6a6

Reply:
That's a good point, thank you